### DISCUSSION TITLE
Minor issue: Missleading German translation in identity field "License Number"

### DISCUSSION THREAD

**Discussion by homecasagit** [dani-garcia/vaultwarden#3194] [Ideas]
### Subject of the issue
Minor issue: Missleading German translation in identity field "License Number"

### Deployment environment

* vaultwarden version: 
2022.12.0

* Install method: 
docker - image: vaultwarden/server:1.27.0

* Clients used: 
web vault, compared to official bitwarden Android app and browser extensions

### Steps to reproduce

1. install vaultwarden via docker
2. set language to German
3. create a new identity

### Expected behaviour

**German translation = "Führerscheinnummer"**

This would be similar to translation in bitwarden clients, at least in android app and browser extensions.

### Actual behaviour

**German translation = "Lizenznummer"**

While this is literal translation of "License Number", it is wrong here.
In German, "Lizenznummer" is used in context of e.g. software licenses. The number of the driver license, which is used in context of persons/identities, is called "Führerscheinnummer".

### Troubleshooting data

Not a big issue, but I guess it would be an easy fix. :)

English text in Vaultwarden web vault:

![grafik](https://user-images.githubusercontent.com/100075768/215332142-2182aafc-c67d-4965-ac85-6c2ce33f613d.png)

German text in Vaultwarden web vault:

![grafik](https://user-images.githubusercontent.com/100075768/215332282-2f360853-a4f6-4ff5-a369-121573983d61.png)

German text in bitwarden clients:

![grafik](https://user-images.githubusercontent.com/100075768/215332424-b61254e3-fec7-428a-b56c-b19a18e87315.png)

**Comment by tessus** [comment]
The translation files are not touched by the vw team, so this issue is with the official Bitwarden web-vault.

You can see here https://github.com/dani-garcia/bw_web_builds/pull/105/files what has been changed from the bw code for the version you are using.

**Comment by homecasagit** [comment]
Thank you very much for the clarification, I was not aware of that.

And thanks to all contributors for your awesome work!
